docs/how-to-guide/how-to-guide.md

```diff
@@ -0,0 +1,358 @@
+EOF
+)
+juju deploy wordpress-k8s --config initial_settings=$WORDPRESS_SETTINGS
+```
+
+You can verify your initial WordPress settings by navigating to ​​the general settings page in
+WordPress(`http://<wordpress-unit-ip>/wp-admin/options-general.php`).
+
+You can also pass in the wordpress-k8s configuration.yaml file with the parameters above. See how
+to pass in a configuration file in the
+[juju docs](https://juju.is/docs/olm/manage-applications#heading--configure-an-application-during-deployment).
+
+### Get initial admin credentials
+
+Run the following command to get the initial admin password that can be used to login at
+`http://<wordpress-unit-ip>/wp-login.php`.
+
+```
+juju run-action wordpress-k8s/0 get-initial-password --wait
+```
+
+The output of the action should look something similar to the following:
+
+```
+unit-wordpress-k8s-0:
+  UnitId: wordpress-k8s/0
+  id: "10"
+  results:
+    password: <password> # password should look something like: ATEqzix_phnAEDJOwEWXuK2CJBXlMmTS9e90aIeH9ys
+  status: completed
+```
+
+You can use the password to login to the admin account in `http://<wordpress-unit-ip>/wp-admin.php`.
+
+Note that if `admin_password` value has been passed in the `initial_settings` configuration, the
+password from the action is invalid.
+
+### Rotate WordPress secrets
+
+To securely update all the WordPress secrets, run the following action.
+
+```
+juju run-action wordpress-k8s/0 rotate-wordpress-secrets –wait
+```
+
+This action will force users to be logged out. All sessions and cookies will be invalidated.
+
+### Install plugins
+
+Start by locating the plugin from the WordPress [plugins page](https://wordpress.org/plugins/).
+Once you’ve located the plugin, the plugin slug is the name of the plugin from the URL of the
+selected theme page. For example, `https://wordpress.org/plugins/akismet/` the plugin slug is
+“akismet” after the “/plugins/” path in the URL. You can now install the plugin using the plugin
+slug with `juju config`.
+
+```
+juju config wordpress-k8s plugins=akismet
+```
+
+To install multiple plugins at once, append more plugins separated by a comma.
+
+```
+juju config wordpress-k8s plugins=akismet,404page
+```
+
+Once the configuration is complete, you can navigate to `http://<wordpress-unit-ip>/wp-admin/plugins.php` to
+verify your new plugin installation.
+
+### Install themes
+
+Start by locating the theme from the WordPress themes page. Once you’ve located the theme, the
+theme slug is the name of the theme from the URL of the selected theme page. For example,
+https://wordpress.org/themes/twentytwentytwo/ the plugin slug is “twentytwentytwo” after the
+“/themes/” path in the URL. You can now install the theme using the theme slug with `juju config`.
+
+```
+juju config wordpress-k8s themes=twentytwentytwo
+```
+
+To install multiple themes at once, append more themes separated by a comma.
+
+```
+juju config wordpress-k8s themes=twentytwentytwo,twentytwentythree
```

Comment:
Same as above, default themes.

Reply:
Replaced the theme slugs to be more generic <plugin-theme>.
Thank you for the suggestion 👍